2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1473,83 +2057,564 @@ Diversi ruoli risultano avere un monte ore minore rispetto a quello preventivato
+
+#figure(
+  table(
+    columns: 3,
+    [*Ruolo*], [*Ore rimanenti*], [*Budget rimanente*],
+    [Responsabile],     [41],     [1230],
+    [Amministratore],     [39],     [780],
+    [Analista],     [49],     [1225],
+    [Progettista],     [29],     [725],
+    [Programmatore],     [165],     [2475],
+    [Verificatore],     [101],     [1515],
+    [Rimanente],     [424],     [7950],
+  ),
+  caption: "Monitoraggio sprint 11"
+)
+
+
+== Sprint 12 dal 21-01-2024 al 28-01-2024
+
+=== Obiettivi raggiunti
+- Colloquio con il Professor Cardin superato con luce "ampiamente verde";
+- Aggiunta di grafici al Piano di Progetto:
+  - suddivisione oraria per ruolo per preventivi e consuntivi.
+
+=== Obiettivi mancati
+- Estensione dei termini di Glossario.
+
+=== Problematiche
+Nessuna.
+
+=== Risoluzioni attuate
+Nessuna.
+
+=== Panoramica dei costi effettivi
+
+#figure(
+  table(
+    columns: 8,
+    [*Membro*], [*Responsabile*], [*Amministratore*], [*Analista*], [*Progettista*], [*Programmatore*], [*Verificatore*], [*Totale*],
+    [Banzato],     [0],     [0],     [0],     [1],     [0],     [0],     [1],
+    [Carraro],     [0],     [0],     [1 (+1)],     [1],     [0],     [0],     [2 (+1)],
+    [Gardin],     [0],     [0],     [0],     [1],     [0],     [0],     [1],
+    [Nardo],     [1],     [0],     [0],     [0],     [0],     [0],     [1],
+    [Oseliero],     [0],     [0],     [0],     [0],     [0],     [1],     [1],
+    [Todesco],     [0],     [1],     [0],     [0],     [0],     [0],     [1],
+    [Zaccone],     [0],     [0],     [0],     [0],     [0],     [1],     [1],
+    [Totale ore],     [1],     [1],     [1 (+1)],     [3],     [0],     [2],     [8 (+1)],
+    [Costo ruolo],     [30],     [20],     [25 (+25)],     [75],     [0],     [30],     [180 (+25)],
+  ),
+  caption: "Prospetto del consuntivo, sprint 12"
+)
+#let data = (
+  ("Responsabile", 1, 1),
+  ("Amministratore", 1, 1),
+  ("Analista", 0, 1),
+  ("Progettista", 3, 3),
+  ("Programmatore", 0, 0),
+  ("Verificatore", 2, 2),
+)
+#let x-coordinates = compute-labels-x-coordinate(data, role-chart-size)
+#let y-coordinates = compute-labels-y-coordinate(data, role-chart-size)
+
+#figure({
+  import draw: *
+  canvas({
+    chart.barchart(..barchart-config, data)
+    let i = 0
+    while(i < data.len()) {
+      content(
+        (x-coordinates.at(i).at(0), y-coordinates.at(i).at(0)),
+        [#data.at(i).at(1)],
+        ..barchart-label-config
+      )
+      content(
+        (x-coordinates.at(i).at(1), y-coordinates.at(i).at(1)),
+        [#data.at(i).at(2)],
+        ..barchart-label-config
+      )
+      i += 1
+    }
+  })},
+  caption: "Suddivisione oraria per ruolo, consuntivo sprint 12",
+  kind: "chart",
+  supplement: "Grafico"
+)
+
+Il totale delle ore lavorate, seppur basso, è stato in gran parte rispettato. Tuttavia, il ruolo di Analista è stato attivato per correggere un errore di forma riscontrato nel calcolo dei codici identificativi dei requisiti.
```

Comment:
riformulerei "Il totale delle ore lavorate, seppur basso, è stato in gran parte rispettato." in "Il totale delle ore di lavoro preventivate è stato in gran parte rispettato."